File: client/Client/Network/Messages/Editor/CursorMovedMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Network.Messages
{
    public class CursorMovedMessage
    {
        public const string TYPE = "CursorMovedMessage";
        private CursorDTO _cursor;

        public CursorMovedMessage(CursorDTO cdto)
        {
            _cursor = cdto;
        }

        public CursorDTO cursor
        {
            get => _cursor;
            set => _cursor = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/CursorMovedMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.model.Cursor;

public class CursorMovedMessage {

	public static final String TYPE = "CursorMovedMessage";
	private Cursor cursor;

	public CursorMovedMessage() {
	}

	public CursorMovedMessage(Cursor cursor) {
		this.cursor = cursor;
	}

	public Cursor getCursor() {
		return cursor;
	}

	public void setCursor(Cursor cursor) {
		this.cursor = cursor;
	}
}
